tool/lib/commands/packages_get.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:cli_util/cli_logging.dart';
+
+import '../model.dart';
+import '../utils.dart';
+
+class PackagesGetCommand extends Command {
+  @override
+  String get name => 'packages-get';
+
+  @override
+  String get description => "Run 'pub get' in all DevTools packages.";
+
+  @override
+  Future run() async {
+    final FlutterSdk sdk = FlutterSdk.getSdk();
+    if (sdk == null) {
+      print('Unable to locate a Flutter sdk.');
+      return 1;
+    }
+
+    final Logger log = Logger.standard();
+
+    final DevToolsRepo repo = DevToolsRepo.getInstance();
+
+    final List<Package> packages = repo.getPackages();
+
+    log.stdout('Running flutter packages get...');
```

Comment:
r/flutter packages get/flutter pub get

When you run `flutter packages get` or `flutter pub get`, flutter tool prints `Running "flutter pub get" in devtools_app...  `

Reply:
updated